packages/core/helper-plugin/lib/src/components/RelationInput/components/Option.js

```diff
@@ -20,12 +20,12 @@ const StyledBullet = styled.div`
 `;
 
 export const Option = (props) => {
+  console.log(props);
   const { formatMessage } = useIntl();
   const Component = components.Option;
-  const hasDraftAndPublish = has(get(props, 'data.value'), 'publishedAt');
+  const hasDraftAndPublish = has(get(props, 'data'), 'isDraft');
```

Comment:
Right now you can't retreive it like that. I could pass you:

```js
isDraft: PropTypes.bool.isRequired,
hasDraftandPublish: PropTypes.bool
```

for each relation?

Reply:
Nevermind: let's make this future compatible:

```js
publicationState: 'draft' | 'published' | false
```

?

Reply:
Done in https://github.com/strapi/strapi/pull/14092/commits/add0c9f54f914204ef3a7d89b2a24402bd5ada0b. I think this will work quite well (see my comment about passing translations ⬆️ )

Reply:
sounds neat - just to be sure I got it, RelationInput will receive new props:
`publicationState: string || bool => 'Draft" || "Published" || false`
and "Draft"-"Published" string will already be translated when arriving?

Reply:
Not quiet: every search result/ relation with have an attribute called `publicationState` (instead of `isDraft`) with the keys I've mentioned. The same keys are used by the prop `publicationStateTranslations` (suggestion) which is passed to `RelationInput`. I've mentioned an example above how these can be used on each relation / search result.

Reply:
I see! works for me :)